### ISSUE TITLE
[SOLUTION] 2주차 문제 풀이 

### ISSUE THREAD

**Issue by myunbongs** [myunbongs/cgvm-winter-study-2023#4]
## 문제 내용
- [ ] [Q3: Implexment a Softmax classifier](https://cs231n.github.io/assignments2021/assignment1/)
- [ ] [Q4: Two-Layer Neural Network](https://cs231n.github.io/assignments2021/assignment1/)
